src/componentManifest.ts

```diff
@@ -104,6 +104,13 @@ export function makeDeviceManifest({ projectConfig, buildId } : {
     },
 
     flush(done) {
+      // Ensure the fallback locale is the first listed in the manifest
+      const { [projectConfig.fallbackLocale]: fallbackLocale, ...otherLocales } = manifest.i18n;
```

Comment:
Should we use a Map here to ensure order of elements? Since there's no guarantee https://stackoverflow.com/a/5525820

Reply:
[ES2015 _does_ guarantee an ordering of Object elements.](https://www.ecma-international.org/ecma-262/6.0/#sec-ordinary-object-internal-methods-and-internal-slots-ownpropertykeys). The spec technically doesn't specify an order for `JSON.stringify`'ing an Object, but in practice browsers and Node serialize the properties in `[[OwnPropertyKeys]]` order. And besides, you can't `JSON.stringify` a `Map` (it serializes to `{}`).